src/lib/Zikula/Bundle/CoreBundle/EventListener/LegacyRouteListener.php

```diff
@@ -85,14 +85,8 @@ public function onKernelRequest(GetResponseEvent $event)
             }
 
             // call the requested/homepage module
-            try {
-                ModUtil::getModule($module);
-                $newType = true;
-            } catch (\Exception $e) {
-                $newType = false;
-            }
-
-            if ($newType) {
+            $moduleBundle = ModUtil::getModule($module);
+            if (null !== $moduleBundle) {
                 $return = ModUtil::func($modinfo['name'], $type, $func);
             } else {
                 $return = ModUtil::func($modinfo['name'], $type, $func, $arguments);
```

Comment:
I wonder if this condition makes sense at all.
Why should we only provide `$arguments` if the module bundle does exist?